File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BasePartyCCType.vb
Public Class BasePartyCCType
    Inherits BasePartyType
    Public Property CompanyName As String
    Public Property BusinessCode As String
    Public Property MainContact As String
    Public Property NumberOfOffices As Integer
    Public Property NumberOfOfficesSpecified As Boolean
    Public Property NumberOfEmployees As String
    Public Property TradeCode As String
    Public Property ClientDetail As BaseClientSharedDataType
    Public Property CompanyReg As String
    Public Property SICCode As String
    Public Property TradingSince As Date
    Public Property TradingSinceSpecified As Boolean
    Public Property WageRoll As Decimal
    Public Property WageRollSpecified As Boolean
    Public Property TurnoverCode As String
    Public Property FinancialYear As Date
    Public Property FinancialYearSpecified As Boolean
    Public Property Salutation As String
    Public Property TPS As Boolean
    Public Property TPSSpecified As Boolean
    Public Property MPS As Boolean
    Public Property MPSSpecified As Boolean
    Public Property EMPS As Boolean
    Public Property EMPSSpecified As Boolean
    Public Property Source As String
    Public Property AlternativeId As String
End Class

File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BasePartyCCType.cs
namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BasePartyCCType : BasePartyType
    {
        public string CompanyName { get; set; } = string.Empty;
        public string BusinessCode { get; set; } = string.Empty;
        public string MainContact { get; set; } = string.Empty;
        public int NumberOfOffices { get; set; }
        public bool NumberOfOfficesSpecified { get; set; }
        public string NumberOfEmployees { get; set; } = string.Empty;
        public string TradeCode { get; set; } = string.Empty;
        public BaseClientSharedDataType ClientDetail { get; set; }
        public string CompanyReg { get; set; } = string.Empty;
        public string SICCode { get; set; } = string.Empty;
        public System.DateTime TradingSince { get; set; }
        public bool TradingSinceSpecified { get; set; }
        public decimal WageRoll { get; set; }
        public bool WageRollSpecified { get; set; }
        public string TurnoverCode { get; set; } = string.Empty;
        public System.DateTime FinancialYear { get; set; }
        public bool FinancialYearSpecified { get; set; }
        public string Salutation { get; set; } = string.Empty;
        public bool TPS { get; set; }
        public bool TPSSpecified { get; set; }
        public bool MPS { get; set; }
        public bool MPSSpecified { get; set; }
        public bool EMPS { get; set; }
        public bool EMPSSpecified { get; set; }
        public string Source { get; set; } = string.Empty;
        public string AlternativeId { get; set; } = string.Empty;
    }
}
